.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2
 jobs:
   build:
```

Comment:
I would like to rename this job from `build` to `test`, do you guys think this is a good idea? `build` is a little misleading, because all it does is test only (at least now).

Reply:
I would agree with that. :+1:

Reply:
Thanks. I will rename it once the current CircleCI deployment is done.